pkg/trace/api/profiles.go

```diff
@@ -57,6 +57,9 @@ func profilingEndpoints(apiKey string) (urls []*url.URL, apiKeys []string, err e
 			}
 		}
 	}
+	for i, u := range urls {
+		log.Debugf("Set profiling intake URL [%d]:%s", i, u)
```

Comment:
Are those scrubs ? URLs might contains password that we don't want in the logs and then in the flares.

Reply:
We send the API key via header in profiling so these URLs should be clean. They have to point directly to profiling intake addresses in different environments (e.g. `intake.profile.datadoghq.com`, `intake.profile.datadoghq.eu`). 

I guess someone could decide to put whatever they wanted on one of those places but if we follow that route then we'd have to potentially scrub all configuration fields.

Reply:
If we were worried about proxy configurations, these go through the same configuration path as APM and are not logged anywhere: https://github.com/DataDog/datadog-agent/blob/6275ef2adaf6dcacff9a3b0630d2e8ecf7762629/pkg/trace/config/config.go#L235

Reply:
I'm worried about people using a proxy with a login+password. We do scrub the login+password when logging them everywhere.

Reply:
Are we talking about using a proxy without actually configuring it as such in the agent but just specifying some weird address in the url? Wasn't aware this was something people did.

But isn't this covered by us using the agent-wide logger which already seems to scrub all logged messages? https://github.com/DataDog/datadog-agent/blob/6c1c5208173f3643b14679964c01a03c2ee2af3f/pkg/util/log/log.go#L179-L180